integration_tests/environment.py

```diff
@@ -72,3 +73,9 @@ def before_all(context):
     constants.D3A_TEST_RUN = True
     if os.environ.get("DISPATCH_EVENTS_BOTTOM_TO_TOP") == "False":
         d3a.constants.DISPATCH_EVENTS_BOTTOM_TO_TOP = False
+    os.system('docker-compose up -d')
+    sleep(60)
```

Comment:
This will delay the integration tests needlessly IMO, because even at the current implementation the integration tests do not use Redis at all, and I think the same should apply for Kafka